src/components/Item/Item.module.css

```diff
@@ -0,0 +1,7 @@
+.item {
+    color: #333;
+}
+
+.done {
+    text-decoration: line-through;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5688965)